tests/markdown-test.el

```diff
@@ -5921,7 +5921,7 @@ Details: https://github.com/jrblevin/markdown-mode/issues/308"
     (markdown-table-align)
     (should (string= (buffer-substring-no-properties (point-min) (point-max))
                      "| Col1 | Col2 |
-| :-:  | :-:  |
+|:----:|:----:|
```

Comment:
Could you add new tests for this fix instead of modifying existed one ?

Reply:
Ok, I've added two tests, one for spaces and another for tabs.